src/main/java/pl/pjwstk/woloappapi/controller/UserController.java

```diff
@@ -43,6 +40,15 @@ public ResponseEntity<List<UserResponseDto>> getUsers() {
 
     }
 
+    @GetMapping("/volunteers")
+    public ResponseEntity<List<UserShortResponse>> getUserShorts() {
+        List<User> users = userService.getUsersOnlyWithUserRole();
```

Comment:
Czy to jest endpoint otwarty na każdego? Czy na innym poziomie gdzieś już jest zdefiniowane, kto powinien móc to widzieć?

Reply:
Jest zdefiniowane - tylko admin ma prawo, w pliku config do security